src/test/java/fr/pilato/elasticsearch/crawler/fs/test/integration/FsCrawlerImplAllParametersIT.java

```diff
@@ -525,6 +525,24 @@ public void test_filename_as_id() throws Exception {
         }), equalTo(true));
     }
 
+    public void test_add_as_inner_object() throws Exception {
```

Comment:
No idea how I could believe that test worked. Should remember to stay off LSD while coding.